packages/core/src/logger.js

```diff
@@ -59,8 +53,11 @@ exports.init = function init(config = {}) {
     // A custom logger which is neither bunyan nor pino has been provided via config. We use it as is.
     parentLogger = config.logger;
   } else {
-    // No custom logger has been provided via config, we create a new minimal logger.
-    parentLogger = consoleLogger;
+    // No custom logger has been provided via config, we create a new minimal logger with pino.
+    // @ts-ignore
+    parentLogger = pino({
```

Comment:
There is indeed a problem with falling back to a "minimal" logger. 
A soon as you require the core module, all required files call `getLogger` on the top of the file. Thats why we have added this fallback. This needs to be refactored away, because it does not respect any log level. I'd suggest to raise an issue for that. But IMO this is not the cause of the problem. 

IMO we are seeing debug logs by default from **any** serverless packages because we do not forward their logger!

See https://github.com/instana/nodejs/blob/v3.17.1/packages/azure-container-services/src/activate.js

We define which logger we want to use for the pkg (logger from serverless pkg).
We normalize the config (which I also do think is very wrong!, but lets ignore that. Please add a comment.) and then we do not forward the logger in the config:

https://github.com/instana/nodejs/blob/v3.17.1/packages/core/src/index.js#L69
https://github.com/instana/nodejs/blob/v3.17.1/packages/core/src/logger.js#L43

The **shortest quick fix** (without having too much to refactor) is IMO for every serverless pkg:

```
 const config = normalizeConfig({});
 config.logger = logger;
```